sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-full-time.integration.service.ts

```diff
@@ -71,8 +71,8 @@ export class ECertFullTimeIntegrationService extends ECertIntegrationService {
       // ! All awards effective values are rounded to the nearest integer (0.5 rounds up).
       // ! studentAmount and schoolAmount have the decimal part combined into the integer part because
       // ! the schoolAmount contains decimals and schoolAmount is used to determine the studentAmount.
-      const studentAmount = disbursementAmount - ecertRecord.schoolAmount;
-
+      const studentAmount =
```

Comment:
No changes needed here if we follow https://github.com/bcgov/SIMS/pull/1840/files#r1152324927